src/java.desktop/share/native/libharfbuzz/hb-utf.hh

```diff
@@ -458,19 +458,21 @@ struct hb_ascii_t
 template <typename utf_t>
 static inline const typename utf_t::codepoint_t *
 hb_utf_offset_to_pointer (const typename utf_t::codepoint_t *start,
```

Comment:
This change is not really necessary: `hb_utf_offset_to_pointer()` is not used anywhere in this project (that I can see), but that also makes it safe to apply.